modules/CompanyUser/Controllers/CompanyUserController.php

```
@@ -104,7 +102,16 @@ public function store(CreateCompanyUserRequest $request)
             $request->createCreateCompanyUserCompanyRoleDTO()
         );
         $presenter = new CompanyUserPresenter($createdItem);
-        return Json::item($presenter->getData());
+
+        // Check if email was sent successfully
+        $message = __('messages.company_user.created');
```

The `email_sent` property is being accessed using null coalescing operator with a default of `true`, which might mask actual issues. Consider:

```php
$emailSent = $createdItem->email_sent !== false;
```

This makes the intention clearer that we're specifically checking for `false` values.